### PR TITLE
Fixes #35246 - Default cv creation error during seed

### PR DESCRIPTION
**Pull request by sjha4**
#### What are the changes introduced in this pull request?
1. Adds a validation to ensure default CVs have Default Content View as name and Default_content_view as label.
2. Ensure db:seed doesn't fail for default CVs with wrong label and corrects the label.

#### What are the testing steps for this pull request?
1. Create a test org
4. In your console: 
cv = Katello::ContentView.where(default: true, organization_id: 4)&.first
cv.update_attribute(:label, "abc")
5. org.update_attribute(:created_in_katello, false) 
6. Run bundle exec rails db:seed
7. Without this change, it should fail.

### CODE REVIEW

**Comment by sjha4** [comment]
[test katello]

**Comment by sjha4** [comment]
[test katello]

**Comment by sjha4** [comment]
[test katello]

**Comment by sjha4** [comment]
[test katello]

File: app/lib/actions/katello/repository/upload_files.rb [hunk 1]
```diff
@@ -29,7 +30,7 @@ def plan(repository, files, content_type = nil, options = {})
             tmp_files.each do |file|
               sequence do
                 upload_action = plan_action(Pulp3::Orchestration::Repository::UploadContent,
-                                                 repository, SmartProxy.pulp_primary!, file, unit_type_id)
+                                            repository, SmartProxy.pulp_primary!, file, unit_type_id)
```
Comment (parthaa) [inline app/lib/actions/katello/repository/upload_files.rb:33]: how is this related to this PR ?

**Comment by parthaa** [comment]
Looks like the only relevant that changed in `models/katello/content_view.rb`  is a new method `check_default_label_name` and a validates on that. Could we not do the formatting in this file, will make it easier to review and cherry pick?

**Review by parthaa — APPROVED** [review]
